bin/scriptLib

```diff
@@ -11,7 +11,7 @@ set -o pipefail
 # CURRENT_BRANCH is used when updating Whitesource to determine the correct project name
 export CURRENT_BRANCH=${TRAVIS_BRANCH}
 
-EXTRA_OPTS=""
+EXTRA_OPTS="-Dakka.test.timefactor=10"
```

Comment:
Oh, we need to apply a `timefactor` to all tests? :-(

Reply:
I don't know if it really needs to be added to all tests, or if it needs to be increased by such a large factor, but I was experiencing several test failures specifically in Travis on this branch, and I thought it couldn't hurt. I could try to find a more optimized value if you think it's worth it, but it's a pretty time-consuming process to retest.

It looks like `master` hasn't had a passing build for a while, but the [build times on this branch](https://travis-ci.com/lagom/lagom/builds/114383197) seem roughly comparable to the times for passing jobs in recent builds:

* https://travis-ci.com/lagom/lagom/builds/114998394
* https://travis-ci.com/lagom/lagom/builds/114876996
* https://travis-ci.com/lagom/lagom/builds/114842956

So, I think this will only impact build times for legitimate failures.